tools/etcd-dump-db/main.go

```diff
@@ -40,6 +41,11 @@ var (
 		Short: "iterate-bucket lists key-value pairs in reverse order.",
 		Run:   iterateBucketCommandFunc,
 	}
+	scanKeySpaceCommand = &cobra.Command{
+		Use:   "scan-keys [data dir or db file path] [start revision]",
+		Short: "scan-keys scans all the key-value pairs starting from a specific revision in the key space.",
```

Comment:
Suggest we also update documentation to explain the new command in https://github.com/etcd-io/etcd/blob/main/tools/etcd-dump-db/README.md

Reply:
Good catch, will update later.

Reply:
Done.